ui/imports/shared/controls/InformationTile.qml

```diff
@@ -17,8 +17,10 @@ Rectangle {
     property alias tagsModel: tags.model
     property alias tagsDelegate: tags.delegate
     property int maxWidth: 0
+    property bool copy: false
+    property var store
```

Comment:
sharing store to a control is not a good idea :(
But I understand that CopyToClipBoardButton needs it, maybe instead of adding this under informationTile we can create another component?

Reply:
Ideally the control would not perform any action but rather emit a signal which can then be handled one (or more) level up

Reply:
yes changing to signal!